docs/userguide/networking/default_network/binding.md

```diff
@@ -0,0 +1,74 @@
+<!--[metadata]>
++++
+title = "Bind container ports to the host"
+description = "expose, port, docker, bind publish"
+keywords = ["Examples, Usage, network, docker, documentation, user guide, multihost, cluster"]
+[menu.main]
+parent = "smn_networking_def"
++++
+<![end-metadata]-->
+
+# Bind container ports to the host
+
+The information in this section explains binding container ports within within the Docker default bridge. This is a `bridge` network named `bridge` created automatically when you install Docker.  
+
+**Note**: The [Docker networks feature](../dockernetworks.md) allows you to create user-defined networks in addition to the default bridge network.
+
+By default Docker containers can make connections to the outside world, but the
+outside world cannot connect to containers.  Each outgoing connection will
+appear to originate from one of the host machine's own IP addresses thanks to an
+`iptables` masquerading rule on the host machine that the Docker server creates
+when it starts:
+
+```
+# You can see that the Docker server creates a
+# masquerade rule that let containers connect
+# to IP addresses in the outside world:
+
+$ sudo iptables -t nat -L -n
+...
+Chain POSTROUTING (policy ACCEPT)
+target     prot opt source               destination
+MASQUERADE  all  --  172.17.0.0/16       0.0.0.0/0
+...
+```
+
+But if you want containers to accept incoming connections, you will need to provide special options when invoking `docker run`. There are two approaches.
+
+First, you can supply `-P` or `--publish-all=true|false` to `docker run` which is a blanket operation that identifies every port with an `EXPOSE` line in the image's `Dockerfile` or `--expose <port>` commandline flag and maps it to a host port somewhere within an _ephemeral port range_. The `docker port` command then needs to be used to inspect created mapping. The _ephemeral port range_ is configured by `/proc/sys/net/ipv4/ip_local_port_range` kernel parameter, typically ranging from 32768 to 61000.
+
+Mapping can be specified explicitly using `-p SPEC` or `--publish=SPEC` option. It allows you to particularize which port on docker server - which can be any port at all, not just one within the _ephemeral port range_ -- you want mapped to which port in the container.
+
+Either way, you should be able to peek at what Docker has accomplished in your network stack by examining your NAT tables.
+
+```
+# What your NAT rules might look like when Docker
```

Comment:
I'm going to STET these. They were in the original. I fixed the first one for ya tho.